Jenkinsfile

```diff
@@ -714,6 +607,9 @@ def testGoTestSuite(prefix, packagesToTest) {
         flags: '-race',
         timeout: '30s',
       ],
+      'github.com/keybase/client/go/systests': [
+        citogo_extra: '-parallel=8',
```

Comment:
cool! teams/chat too? or were there races?

Reply:
I can try those in a separate PR

Reply:
Actually I'll revert this change and try all of those in a separate PR.